cgnsutilities/cgnsutilities.py

```diff
@@ -51,15 +51,22 @@ def __init__(self):
         self.name = "domain"
         self.cellDim = 3
 
-    def printInfo(self):
-        """Print some information on the mesh to screen. Specifically
-        information needed by the drag prediction workshop"""
+    def getCellsNodes(self):
+        """ Returns the total number of Cells and Nodes in the grid. """
         totalCells = 0
         totalNodes = 0
         for blk in self.blocks:
             totalCells += (blk.dims[0] - 1) * (blk.dims[1] - 1) * (blk.dims[2] - 1)
             totalNodes += blk.dims[0] * blk.dims[1] * blk.dims[2]
 
+        return totalCells, totalNodes
+        
+
+    def printInfo(self):
+        """Print some information on the mesh to screen. Specifically
+        information needed by the drag prediction workshop"""
```

Comment:
Since this is being updated, please update the docstring with more relevant text.